src/features/section/queries.ts

```diff
@@ -0,0 +1,39 @@
+import { useQuery, useQueryClient, useMutation } from '@tanstack/react-query';
+import sectionsApis from './apis';
+const sectionKeys = {
+  all: ['sections'] as const,
+  lists: () => [...sectionKeys.all, 'list'] as const,
+};
+const sectionsQueries = {
+  read: () => {
+    return useQuery({
+      queryKey: sectionKeys.lists(),
+      queryFn: () => sectionsApis.get(),
+    });
+  },
+
+  create: () => {
+    const queryClient = useQueryClient();
+    return useMutation({
+      mutationFn: sectionsApis.post,
+      onSuccess: () => {
+        queryClient.invalidateQueries({
+          queryKey: sectionKeys.lists(),
+        });
+      },
+    });
+  },
+
+  delete: () => {
+    const queryClient = useQueryClient();
+    return useMutation({
+      mutationFn: sectionsApis.delete,
+      onSuccess: () => {
+        queryClient.invalidateQueries({
+          queryKey: sectionKeys.lists(),
+        });
+      },
+    });
+  },
+};
+export default sectionsQueries;
```

Comment:
```suggestion
import { useQuery, useQueryClient, useMutation } from '@tanstack/react-query';
import sectionsApis from './apis';

const sectionKeys = {
  all: ['sections'] as const,
  lists: () => [...sectionKeys.all, 'list'] as const,
};

const sectionsQueries = {
  read: () => {
    return useQuery({
      queryKey: sectionKeys.lists(),
      queryFn: () => sectionsApis.get(),
    });
  },
  create: () => {
    const queryClient = useQueryClient();
    return useMutation({
      mutationFn: sectionsApis.post,
      onSuccess: () => {
        queryClient.invalidateQueries({
          queryKey: sectionKeys.lists(),
        });
      },
    });
  },
  delete: () => {
    const queryClient = useQueryClient();
    return useMutation({
      mutationFn: sectionsApis.delete,
      onSuccess: () => {
        queryClient.invalidateQueries({
          queryKey: sectionKeys.lists(),
        });
      },
    });
  },
  
};

export default sectionsQueries;
```
이런 코드들이 import, key 객체, query 객체, export 부분 4영역으로 분리되어야 합니다 ㅎㅎㅎ